client/packages/system/src/Stock/Components/NewStockLineModal.tsx

```diff
@@ -131,6 +131,8 @@ export const NewStockLineModal = ({
                     dosesPerUnit: newItem.doses,
                   },
                   packSize: newItem.defaultPackSize,
```

Comment:
Related side question bc of checking pack-to-one - should stocktake and this modal account for pack to 1 pref? Currently they just use default pack size 👀

Reply:
It converts in backend and think better to do the changes for that [here](https://github.com/msupply-foundation/open-msupply/issues/5761), since when you transfer you still receive > 1 pack size